src/com/facebook/buck/cxx/CxxTest.java

```diff
@@ -63,13 +62,9 @@
 public abstract class CxxTest extends AbstractBuildRuleWithDeclaredAndExtraDeps
     implements TestRule, HasRuntimeDeps, BinaryBuildRule {
 
-  @AddToRuleKey private final ImmutableMap<String, Arg> env;
-  @AddToRuleKey private final ImmutableList<Arg> args;
-  @AddToRuleKey private final Tool executable;
-
-  @AddToRuleKey
-  @SuppressWarnings("PMD.UnusedPrivateField")
-  private final ImmutableSortedSet<? extends SourcePath> resources;
```

Comment:
We cannot remove this field right now. Some tests use resources and removing this filed would break situations when a resource has changed but the test's rule key hasn't changed.

The proper handling of these resources would be to provide access to them from test's current folder (by symlinking like in genrule), but right now we don't have this logic.

Reply:
I can add this back and keep it adding to the rulekey